data/bellsofaba.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Jack Aba",
+  "displayStatsPublic": true,
```

Comment:
The displayStatsPublic, avatar and type tags are no longer required, take a look at the [QuickStart Doc.](https://linkfree.io/docs/quickstart-json) if you haven't already.
![screen](https://github.com/EddieHubCommunity/LinkFree/assets/19821445/af33f593-8e59-4e5a-9739-2b54e47caf32)

```suggestion
```